internal/scti/chain_validation.go

```diff
@@ -149,7 +149,7 @@ func isPrecertificate(cert *x509.Certificate) (bool, error) {
 // supplied in the chain. Then applies the RFC requirement that the path must involve all
 // the submitted chain in the order of submission.
 // TODO(phboneff): make this a method func([][]byte) ([]*x509.Certificate, error)
-func validateChain(rawChain [][]byte, validationOpts ChainValidationOpts) ([]*x509.Certificate, error) {
+func (opts ChainValidationOpts) validateChain(rawChain [][]byte) ([]*x509.Certificate, error) {
```

Comment:
It sounds weird to add the validation function to the `ChainValidationOpts` type which is supposed to store the options.

Reply:
Ah sorry I had not seen this comment! Your comment is on point: it's actually the next pr in the chain: https://github.com/phbnf/static-ct/compare/functionsmove...phbnf:static-ct:interfaceverifier. This next PR actually goes further and defines an interface. I did not send it in this PR to make reviewing easier: this one just moves methods, then I'll define interface and rename everything.

Reply:
Great. I'm looking forward to reviewing the next PR.